src/bundle/Resources/views/themes/admin/content/page_title_edit.html.twig

```diff
@@ -4,7 +4,7 @@
     {% if without_close_button is not defined or without_close_button != true %}
         <a
             class="ez-content-edit-container__close"
-            href="{{ path('_ez_content_view', { 'contentId': parent_location.contentId, 'locationId': parent_location.id}) }}"
+            href="{{ path('_ez_content_view', { 'contentId': max(parent_location.contentId, 1), 'locationId': max(parent_location.id, 2)}) }}"
```

Comment:
I think it should be done in a different way. In edit mode, it should get the location not the parent location.
Please see how it was in 3.1
https://github.com/ezsystems/ezplatform-admin-ui/blob/2.1/src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig#L16

Reply:
Done in https://github.com/ezsystems/ezplatform-admin-ui/pull/1568/commits/7d64bddb70506ef426d8fc60286d85dd21d02736